prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -723,6 +726,23 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             ?.getValue(DSL.max(SETTING.CREATED_AT))
     }
 
+    fun saveRemoteConnectionCheck(
+        txn: DataAccessTransaction? = null,
+        connectionCheck: CheckFunction.RemoteConnectionCheck
+    ) {
+        val ctx = if (txn != null) DSL.using(txn) else create
+        val initiatedOn = connectionCheck.initiatedOn.atOffset(ZoneOffset.UTC)
+        val completedOn = connectionCheck.initiatedOn.atOffset(ZoneOffset.UTC)
```

Comment:
This should probably be `connectionCheck.completedOn`

Reply:
Good catch. Thx

Reply:
Done